conda_devenv/devenv.py

```diff
@@ -2,24 +2,61 @@
 
 import argparse
 import os
+import re
 import sys
 
 import six
 
 
+def preprocess_selector_in_line(line):
+    pattern = r"(.*?)#\s+(\[)(.*)(\])(.*)"
+    x = re.search(pattern, line)
+    if x is None:
+        return line
+    if x.lastindex != 5:
```

Comment:
I don't think we need this check, because unless I'm mistaken we either will match the line fully or not, or am I missing something?

Reply:
You're right. I added that just as a safeguard for strange things but I guess we don't need it.